gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/JoinKnoxShellTableBuilder.java

```diff
@@ -52,6 +52,12 @@ public KnoxShellTable on(String columnName) {
     return on(leftIndex, rightIndex);
   }
 
+  public KnoxShellTable on(String columnName, String columnName2) {
```

Comment:
nit: if I were you I'd use `columnNameLeft` and `columnNameRight` argument names